tensorflow/core/util/gpu_launch_config.h

```diff
@@ -138,12 +138,12 @@ inline GpuLaunchConfig GetGpuLaunchConfig(int work_element_count,
                d.getNumCudaMultiProcessors());
```

Comment:
definitely could be merged together now as APIs in Eigen have be consolidated.

Reply:
you are right...will combine the two